src/test/java/api/requests/scenarios/MoveRequestPolicyTests.java

```diff
@@ -0,0 +1,419 @@
+
+    requestByJames = requestsClient.get(requestByJames);
+    assertThat(requestByJames.getJson().getString(REQUEST_TYPE), is(RequestType.RECALL.getValue()));
+    assertThat(requestByJames.getJson().getInteger("position"), is(1));
+    assertThat(requestByJames.getJson().getString("itemId"), is(interestingTimes.getId().toString()));
+
+    // check item queues are correct size
+    MultipleRecords<JsonObject> smallAngryPlanetQueue = requestsFixture.getQueueFor(smallAngryPlanet);
+    assertThat(smallAngryPlanetQueue.getTotalRecords(), is(1));
+
+    MultipleRecords<JsonObject> interestingTimesQueue = requestsFixture.getQueueFor(interestingTimes);
+    assertThat(interestingTimesQueue.getTotalRecords(), is(1));
+  }
+
+  @Test
+  public void moveRecallRequestWithoutExistingRecallsAndWithNoPolicyValuesChangesDueDateToSystemDate()
+      throws InterruptedException,
+      ExecutionException,
+      TimeoutException,
+      MalformedURLException {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource interestingTimes = itemsFixture.basedUponInterestingTimes();
+    final IndividualResource steve = usersFixture.steve();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    // steve checks out smallAngryPlanet
+    final IndividualResource loan = loansFixture.checkOutByBarcode(
+      smallAngryPlanet, steve, DateTime.now(DateTimeZone.UTC));
+
+    final String originalDueDate = loan.getJson().getString("dueDate");
+
+    // charlotte checks out interestingTimes
+    loansFixture.checkOutByBarcode(interestingTimes, charlotte);
+
+    // jessica places recall request on interestingTimes
+    IndividualResource requestByJessica = requestsFixture.placeHoldShelfRequest(
+      interestingTimes, jessica, DateTime.now(DateTimeZone.UTC), RequestType.RECALL.getValue());
+
+    assertThat(patronNoticesClient.getAll().size(), is(0));
+
+    // move jessica's recall request from interestingTimes to smallAngryPlanet
+    IndividualResource moveRequest = requestsFixture.move(new MoveRequestBuilder(
+      requestByJessica.getId(),
+      smallAngryPlanet.getId(),
+      RequestType.RECALL.getValue()
+    ));
+
+    assertThat("Move request should have correct item id",
+      moveRequest.getJson().getString("itemId"), is(smallAngryPlanet.getId().toString()));
+
+    assertThat("Move request should have correct type",
+      moveRequest.getJson().getString("requestType"), is(RequestType.RECALL.getValue()));
+
+    final JsonObject storedLoan = loansStorageClient.getById(loan.getId()).getJson();
+
+    assertThat("due date is the original date",
+      storedLoan.getString("dueDate"), not(originalDueDate));
+
+    final String expectedDueDate = ClockManager.getClockManager().getDateTime().toString(ISODateTimeFormat.dateTime());
```

Comment:
There has to be a better way to do this, but it is likely a change for another day